deny.toml

```diff
@@ -109,6 +109,7 @@ allow = [
 	# NOTE: not to be confused with business source license!
 	"BSL-1.0",
 	"Unicode-3.0",
+	"CDLA-Permissive-2.0",
```

Comment:
What package that we are using requires this?

Reply:
Webpki-roots, I think. You can see the failing pipeline from the first commit: https://github.com/wasmerio/wasmer/actions/runs/16301920672/job/46038415638